src/Microsoft.AspNet.Mvc.ModelBinding/ValueProviders/FormValueProviderFactory.cs

```diff
@@ -19,8 +20,8 @@ public IValueProvider GetValueProvider([NotNull] ValueProviderFactoryContext con
             if (IsSupportedContentType(request))
             {
                 var culture = GetCultureInfo(request);
-                return new ReadableStringCollectionValueProvider<IFormDataValueProviderMetadata>(
```

Comment:
It is used only in `QueryStringValueProviderFactory`. Not sure what I should rename it to.